platform/security/idp/security-idp-client/src/main/resources/templates/soap.handlebars

```diff
@@ -12,10 +11,8 @@
  *
  **/
 -->
-<soapenv:Envelope
-        xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"
-        xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
❗️ I think we still need to add these (and the ones in the idp-server soap template) back in

Reply:
Ok. I put them back int. Those attributes were missing on `soapfault.handlebars` file on master branch, so I added them to `soapfault.handlebars` for consistency.